modules/nf-core/mitohifi/mitohifi/main.nf

```diff
@@ -37,16 +38,16 @@ process MITOHIFI_MITOHIFI {
     script:
     // Exit if running this module with -profile conda / -profile mamba
     if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
-        exit 1, "MitoHiFi module does not support Conda. Please use Docker / Singularity instead."
+        error "MitoHiFi module does not support Conda. Please use Docker / Singularity instead."
     }
 
     def args = task.ext.args ?: ''
-    def args2 = task.ext.args2 ?: ''
-    def script_input = args2.equals("-r") ? "-r ${input}" :
-                args2.equals("-c") ? "-c ${input}" :
-                exit("-r for reads or -c for contigs must be specified")
+
+    if ( ! input_mode or ! (input_mode in [ '-r', '-c' ]) ) {
```

Comment:
```suggestion
    if ( input_mode !in [ '-r', '-c' ] ) {
```

Reply:
The merge from `master` brought an alternative way of doing this, via `contains`